test/test_common.py

```diff
@@ -6,7 +6,7 @@
 _release_versions = [
     54, # last release for GRCh36/hg18
     75, # last release for GRCh37/hg19
-    77, # most recent release for GRCh38
+    78, # most recent release for GRCh38
```

Comment:
My plan is to test the latest ensembl release for each reference genome. Otherwise, the tests would run for a very long time if we enumerate all the releases. I guess we also make a rule like "test the past 3 releases".